diff --git a/dist/runtime/emit-code.js b/dist/runtime/emit-code.js
index 523050e..7fab71d 100644
--- a/dist/runtime/emit-code.js
+++ b/dist/runtime/emit-code.js
@@ -100,7 +100,7 @@ const getFunctionName = (fn) => {
  * convert a value to a stringified version
  */
 const valueToString = (value) => {
-    if (typeof value === 'object' && value[Symbol.for('svelte.snippet')]) {
+    if (typeof value === 'object' && value !== null && value[Symbol.for('svelte.snippet')]) {
         return 'snippet';
     }
     if (typeof value === 'function') {
