diff --git a/dist/runtime/emit-code.js b/dist/runtime/emit-code.js
index 33f83f425322e8ebcf7020dea05c8c6b42a4ab0e..e0d1420e869c9ff141d02d6bf9fbc9a323ca3dcb 100644
--- a/dist/runtime/emit-code.js
+++ b/dist/runtime/emit-code.js
@@ -79,7 +79,7 @@ const getFunctionName = (fn) => {
  * convert a value to a stringified version
  */
 const valueToString = (value) => {
-    if (typeof value === 'object' && value[Symbol.for('svelte.snippet')]) {
+    if (typeof value === 'object' && value !== null && value[Symbol.for('svelte.snippet')]) {
         return 'snippet';
     }
     if (typeof value === 'function') {
